docker/docker-compose.yml

```diff
@@ -60,17 +60,17 @@ services:
       timeout: 2s
       retries: 20
 
-  cockroachdb:
```

Comment:
You'll need to undo the changes done here below to make tests pass in GitHub Actions.